deptofdefense/move.mil
# Grab Bag: meta-stuff, nginx config improvements, graphics optimization

## Checklist

I have…


## Summary of Changes

This pull request…

- moves `app/assets/documents` files to `public/documents`,
- moves graphics files in `public` to `app/assets/meta` so that they can be run through Asset Pipeline (should we ever need to replace or update them),
- moves `manifest.json` and `browserconfig.xml`, converting them to Builder-style templates so that we can call assets via Asset Pipeline,
- simplifies the nginx rules around caching static content in `/assets`, and
- adds the HSTS header (thanks, @hlieberman-gov!).

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout rework-meta-things`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails server`,
1. load up http://localhost:3000 and observe that everything works as it always has!

The obvious URLs to check out are http://localhost:3000/browserconfig.xml and http://localhost:3000/manifest.json.
